pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: NutanixCluster
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  prismCentral:
+    address: "{{.nutanixEndpoint}}"
+    port: {{.nutanixPort}}
+    insecure: {{.nutanixInsecure}}
```

Comment:
Is there a way to use TLS even with self-signed certs instead of `insecure`? 
Doesn't have to block this PR but just curious if this is something that can be handled without skipping TLS by passing the CA cert in the config

Reply:
That's the plan, Abhinav! However, at the moment, CAPX doesn't support taking in a additional trust bundle. EKS-A codebase already has the wiring needed for whenever CAPX will support an additional trust bundle. Hopefully in the coming weeks.